docs/content/reference/field.md

```diff
@@ -98,6 +98,22 @@ Example:
 sensitive: true
 ```
 
+### `write_only`
+If true, the field is considered "write-only", which means that its value will
+be obscured in Terraform output as we all as not be stored in state. This field is meant to replace `sensitive` as it doesn't store the value in state.
+See [Ephemerality in Resources - Use Write-only arguments](https://developer.hashicorp.com/terraform/language/resources/ephemeral/write-only)
+for more information.
+
```

Comment:
```suggestion

Write-only fields are only supported in Terraform v1.11+. Because the provider supports earlier Terraform versions, write only fields must be paired with (mutually exclusive) `sensitive` fields covering the same functionality for compatibility with those older versions.
```